src/analysis/normed/order/basic.lean

```diff
@@ -54,6 +55,12 @@ extends linear_ordered_field α, has_norm α, metric_space α :=
 (dist_eq : ∀ x y, dist x y = ‖x - y‖ . obviously)
 (norm_mul' : ∀ x y : α, ‖x * y‖ = ‖x‖ * ‖y‖)
 
+/-- A `normed_lattice_field` is a `normed_linear_ordered_field` that satisfies also the `solid`
+property: `|a| ≤ |b|` implies `‖a‖ ≤ ‖b‖`. -/
+class normed_lattice_field (α : Type*)
+extends normed_linear_ordered_field α :=
+(solid: ∀ a b : α, |a| ≤ |b| → ‖a‖ ≤ ‖b‖)
```

Comment:
```suggestion
(solid : ∀ a b : α, |a| ≤ |b| → ‖a‖ ≤ ‖b‖)
```

Reply:
As you suggested on Zulip, I also added an alias for the lemma solid.